resources/views/herd/show.blade.php

```diff
@@ -19,39 +19,113 @@
                         @include('herd._stats')
                     </div>
                 </div>
-                <div class="card mb-3 herd-card">
-                    <ul class="list-group list-group-flush">
-                        @foreach($elephpants as $key => $elephpant)
-                            <li data-controller="counter"
-                                data-counter-id="{{ $elephpant->id }}"
-                                class="list-group-item {{ $key % 2 === 0 ? 'bg-white' : 'bg-light' }} {{ isset($userElephpants[$elephpant->id]) ? 'has-elephpants' : '' }}">
-                                <div class="row align-items-center no-gutters">
-                                    <div class="col-xl-9 col-lg-8">
-                                        <div class="float-left mr-3">
-                                            <div class="media">
-                                                <a href="javascript:void(0)" data-toggle="popover" data-content="<img src='{{ asset('storage/elephpants/' . $elephpant->image) }}' width='150'/>" data-placement="left" data-trigger="hover">
-                                                    <img src="{{ asset('storage/elephpants/' . $elephpant->image) }}" width="50" alt="" class="img-thumbnail rounded img-fluid mr-3">
-                                                </a>
-                                                <div class="media-body">
-                                                    <p class="mb-0"><strong>{{ $elephpant->name }}</strong> - {{ $elephpant->description }}</p>
-                                                    <p class="mb-0 text-black-50">
-                                                        Sponsor: {{ $elephpant->sponsor }}
-                                                    </p>
+                <ul class="nav nav-tabs" id="myTab" role="tablist">
+                    <li class="nav-item">
+                        <a
+                            class="nav-link active"
+                            id="all-tab"
+                            data-toggle="tab"
+                            href="#all"
+                            role="tab"
+                            aria-controls="all"
+                            aria-selected="true"
+                        >All</a>
```

Comment:
just trying to give users more descriptive names what do you think about `User's elePHPants` here and the `Diff` changed to `ElePHPants I don't have` any idea?